### PR TITLE
Takeover of @GrantBartlett commit - Part 2

### PR DESCRIPTION
**Pull request by BreakSecurity**
Related to https://github.com/tastyigniter/ti-theme-orange/pull/26

and

https://github.com/tastyigniter/ti-theme-orange/pull/22

### CODE REVIEW

File: components/cartbox/mobile.blade.php [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <div
     id="cart-mobile-buttons"
-    class="{{ !$pageIsCheckout ? 'fixed-bottom' : 'mt-3' }}{{ $pageIsCart ? 'hide' : ' d-block d-sm-none' }}"
+    class="{{ !$pageIsCheckout ? 'fixed-bottom' : 'mt-3' }}{{ $pageIsCart ? 'hide' : ' d-block d-lg-none' }}"
```
Comment (sampoyigi) [inline components/cartbox/mobile.blade.php:3]: This should be `d-md-none`

File: components/cartbox/container.blade.php [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="{{ $pageIsCart ?: 'affix-cart d-none d-sm-block' }}">
+<div class="{{ $pageIsCheckout ? 'd-block' : 'affix-cart d-lg-block' }}">
```
Comment (sampoyigi) [inline components/cartbox/container.blade.php:1]: Should be `{{ $pageIsCart ?: 'affix-cart d-none d-md-block' }}`

**Review by sampoyigi — CHANGES_REQUESTED** [review]
Thank you for picking this up!

**Comment by BreakSecurity** [comment]
Done @sampoyigi

File: components/cartbox/mobile.blade.php [hunk 3]
```diff
@@ -1,9 +1,8 @@
 <div
     id="cart-mobile-buttons"
-    class="{{ !$pageIsCheckout ? 'fixed-bottom' : 'mt-3' }}{{ $pageIsCart ? 'hide' : ' d-block d-sm-none' }}"
+    class="{{ !$pageIsCheckout ? 'fixed-bottom' : 'mt-3' }}{{ $pageIsCart ? 'hide' : ' d-block d-md-none' }}"
 >
     @if ($pageIsCheckout)
```
Comment (BreakSecurity) [inline components/cartbox/mobile.blade.php:5]: Should we get rid of this empty if?

Comment (sampoyigi) [reply components/cartbox/mobile.blade.php:5]: Which one?

**Comment by sampoyigi** [comment]
Thank you @BreakSecurity! @GrantBartlett's approach with `lg` looks cleaner.

**Review by sampoyigi — APPROVED** [review]

**Pull request merged**